src/Microsoft.AspNetCore.Mvc.Core/ProducesResponseTypeAttribute.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.Formatters;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    /// <summary>
+    /// Specifies the the type of the value and status code returned by the action
+    /// which can be used to select a formatter while executing <see cref="ObjectResult"/>.
```

Comment:
Please don't mention anything about selecting a formatter here - this attribute has no effect.